client-misk/src/main/kotlin/app/cash/backfila/client/misk/embedded/Backfila.kt

```diff
@@ -11,14 +10,14 @@ import okio.ByteString
  * the Backfila dashboard UI.
  */
 interface Backfila {
-  fun <Type : Backfill<*, *, *>> createDryRun(
+  fun <Type : Any> createDryRun(
```

Comment:
same strangeness here.. why not have a top Backfill interface that DynamoBackfill, HibernateBackfill etc implement?

Reply:
So they can do radically different things. Have different base classes.